src/user/proposals/UserProposalDashboard/DashboardRightPanel/Notifications/Notifications.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from "react";
+import "./OtherIdeas.scss";
+import { ListGroup, Image } from "react-bootstrap";
+import userIcon2 from "../../../../../images/userIcon2.jpg";
+import socket from "../Utils/socket";
+import notifications from "../../../../dashboard/notifications/notifications";
+
+class Notifications extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      socket: socket,
+      token:
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI1ZWRjYjE4ZjIxNWRhNzRjMThkM2YyNzQiLCJpYXQiOjE1OTE1MjE2Nzl9.q3g5Ah_rtjPIrH7z183fVmUBTv_A4OjEoL673zeG250",
+      userId: "5edcb18f215da74c18d3f274",
+      notifications: [],
+    };
+  }
+
+  componentDidMount() {
+    this.fetchNotifications();
+
+    this.state.socket.on("new proposal created", (data) => {
+      data.newNotification = true;
+      data.createdAt = new Date();
+      this.setState({
+        notifications: [...this.state.notifications, data],
+      });
+    });
+
+    this.state.socket.on("proposal deleted", (data) => {
+      data.newNotification = true;
+      data.createdAt = new Date();
+      this.setState({
+        notifications: [...this.state.notifications, data],
```

Comment:
new notifications should be on the top, please check if is this breaking my code functionality